SL/Controller/DeliveryOrder.pm

```diff
@@ -1259,9 +1261,12 @@ sub action_transfer_stock_default {
         $self->js->error(t8('No standard bin set for #1.', $part->displayable_name));
       }
       if ($parts_errors{$part_id}{missing_qty}) {
+        my $bin = SL::DB::Manager::Bin->find_by(
+          id => $parts_errors{$part_id}{bin_id}
+        );
         $self->js->error(
           t8('There are #1 of "#2" missing from the standard bin #3 for transfer.',
-            $parts_errors{$part_id}{missing_qty}, $part->displayable_name, $part->bin->full_description));
+            $parts_errors{$part_id}{missing_qty}, $part->displayable_name, $bin->full_description));
```

Comment:
Wenn ich das richtig verstehe, wird jetzt in der Fehlermeldung der gewählte Lagerplatz gezeigt und nicht der Standardlagerplatz. Ist dann der Text der Fehlermeldung noch richtig?

Reply:
Ne, die ist nicht mehr korrekt. Hab sie angepasst.